src/styles/blocks/content.scss

```diff
@@ -0,0 +1,38 @@
+.content {
+
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax($box-height, 1fr));
+
+  @media (min-width: 900px) {
+    grid-template-columns: repeat(3, 1fr);
+    grid-template-rows: repeat(2,$box-height);
+  }
+
+  @each $index, $col in $c-red {
+    &__box:nth-child(#{$index})::after {
+      background-color: #{$col};
+    }
+  }
+
+  @for $i from 1 through 6 {
+    &__box:nth-child(#{$i})::after {
+      content: "#{$i}";
+      position: absolute;
+      height: $box-height;
+      display: grid;
```

Comment:
You don't need grid for centering content

Reply:
if it's ok - using display:flex;